src/Compiler/Compiler.cpp

```diff
@@ -70,22 +72,29 @@ extern const char* foedag_version_number;
 extern const char* foedag_git_hash;
```

Comment:
This could be remove also foedag_version_number/foedag_git_hash/foedag_build_type